src/platforms/common/performance/index.mdx

```diff
@@ -89,7 +90,7 @@ Learn more about how the options work in <PlatformLink to="/configuration/sampli
 
 ## Verify
 
-<PlatformSection supported={["react-native", "java.spring", "java.spring-boot", "android", "javascript", "apple", "dart", "rust"]} >
+<PlatformSection supported={["react-native", "java.spring", "java.spring-boot", "android", "javascript", "apple", "dart", "rust"]} notSupported={["node"]} >
```

Comment:
Instead of making this `notSupported`, can we restructure the performance docs for Node to have the `automatic-instrumentation` and `custom-instrumentation` sections like we do in JS (https://docs.sentry.io/platforms/javascript/performance/instrumentation/) and Android (https://docs.sentry.io/platforms/android/performance/instrumentation/)

This also means we can add custom performance metric docs to Node as well following that format. We can do this in a follow-up PR though.

Reply:
@onurtemizkan https://github.com/getsentry/sentry-docs/pull/5760 is blocked on this change - so let's try to include that here.